LeaderboardBackend/Controllers/RunsController.cs

```diff
@@ -13,15 +16,17 @@ namespace LeaderboardBackend.Controllers;
 public class RunsController(
     IRunService runService,
     ICategoryService categoryService,
-    IUserService userService
+    IUserService userService,
+    IOptions<JsonOptions> options
     ) : ApiController
 {
     [AllowAnonymous]
+    // We expect an encoded GUID and not the GUID itself. Refer to GuidExtensions.cs.
```

Comment:
Does this note need to be here? This is true for every endpoint that uses a GUID.

Reply:
Fair enough. Speaking of; I realised that we don't respond with encoded GUIDs. I've made https://github.com/leaderboardsgg/leaderboard-backend/issues/298 to cover this case.